styles.css

```diff
@@ -76,17 +76,13 @@ flex-grow: 1;
 
 #coverimg {
   border-radius: 30px 190px 80px 190px;
-  box-shadow: 0px 0px 30px 1px;
+  box-shadow: 0px 0px 30px rgb(37, 19, 58);
   display: flex;
   margin: auto;
   width: 100%;
   height: auto;
 }
 
-figcaption {
```

Comment:
maybe you could add `margin: 2rem` if you like so that the `figcaption` keeps a little distance from the image 😊

Reply:
Thank you! 😊